### DISCUSSION TITLE
New Driver  -  new sections

### DISCUSSION THREAD

**Discussion by exenbema** [Louisvdw/dbus-serialbattery#444] [General]
hello:)

i dont find it in the description - is there a readme, where every part (what we can adjust in the utils file)  is described what this setting is doing?
thx markus

**Comment by mr-manuel** [comment]
Check the new config file which is part of the next version: https://github.com/Louisvdw/dbus-serialbattery/blob/11676e66b3ae1313b9a58bc778f4d6a97fdd8c12/etc/dbus-serialbattery/config.default.ini
